### ISSUE TITLE
Known Issue: Reading json config file fails

### ISSUE THREAD

**Issue by rowan-mcalpin** [RiftEngine/Rift#1]
Call stack:
```
RiftEngine.ConfigFileNotFoundException
  HResult=0x80131500
  Message=The file path "config.json" was not a valid json file.
  Source=Rift
  StackTrace:
   at RiftEngine.GameConfigManager.ReadGameConfiguration(String configFilePath)
   at Example01.Example01.Main() in D:\Users\REMCodes\Documents\Programming\CSharp\RiftExample01\RiftExample01\Program.cs:line 21

  This exception was originally thrown at this call stack:
    Microsoft.Extensions.Configuration.ConfigurationBinder.CreateInstance(System.Type)
    Microsoft.Extensions.Configuration.ConfigurationBinder.BindInstance(System.Type, object, Microsoft.Extensions.Configuration.IConfiguration, Microsoft.Extensions.Configuration.BinderOptions)
    Microsoft.Extensions.Configuration.ConfigurationBinder.Get<T>(Microsoft.Extensions.Configuration.IConfiguration, System.Action<Microsoft.Extensions.Configuration.BinderOptions>)
    RiftEngine.GameConfigManager.ReadGameConfiguration(string)

Inner Exception 1:
InvalidOperationException: Cannot create instance of type 'RiftEngine.Config' because it is missing a public parameterless constructor.
```

**Comment by rowan-mcalpin** [comment]
We are currently investigating this issue.

**Comment by rowan-mcalpin** [comment]
This bug was resolved in [Version  0.0.3](https://www.nuget.org/packages/Rift/0.0.3)
